src/content/docs/dropins/payment-services/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Payment Services overview
+description: Learn about the features of the Payment Services drop-in component.
+---
+
+import Link from '@components/Link.astro';
+import List from '@components/List.astro';
+
+The Payment Services drop-in component provides multiple payment options, such as debit card and credit card fields, digital wallets and PayPal products and services to customize the methods available for your store (or multistore) customers.
```

Comment:
You need to remove the unsupported payment types from the docs. The external developer doesn't know that we intend to create containers with different names. You should also explicitly state in multiple places that credit cards the only supported payment method. I'm not sure if credit cards includes debit cards.

Reply:
works with credit and debit